net-libs/libwebsockets/libwebsockets-3.0.0.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="C library for lightweight websocket clients and servers"
+DESCRIPTION="Libwebsockets (LWS) is a flexible, lightweight pure C library \
+for implementing modern network protocols easily with a tiny footprint, \
+using a nonblocking event loop."
+HOMEPAGE="https://libwebsockets.org/"
+COPYRIGHT="2010-2018 Andy Green"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/warmcat/libwebsockets/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a6b611c212c52f161f70556339fdaa199b7e9b6a167c4638e086d19db75d6290"
+SOURCE_FILENAME="libwebsockets-$portVersion.tar.gz"
+SOURCE_DIR="libwebsockets-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
Shall we mark x86_gcc2 as broken?
(I guess no other recipe will ever need libwebsockets for x86_gcc2 primary arch.)
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```
Hmm, I have just built the recipe for x86 primary arch, so you can mark it as tested if you wish.
(But this is not very usefull because x86 primary arch is not supported anymore, so it would only benefit to users who build their Haiku/x86 themselves.)